libs/components/filter-bar/src/lib/modules/filter-bar/models/filter-bar-filter-modal-opened-args.ts

```diff
@@ -0,0 +1,11 @@
+import { Observable } from 'rxjs';
+
+/**
+ * Arguments passed when a filter modal is opened.
```

Comment:
```suggestion
 * Arguments passed to a filter modal when it opens.
```
Please disregard if I'm incorrectly changing the meaning here. But I assume the arguments are passed to the filter modal, and if that's the case, I think "passed to" is more clear and direct than "passed when," which leaves more room for misinterpretation. But let me know if I'm not following this correctly and got it wrong.

Reply:
addressed in new file